diagnostics/mem.go

```diff
@@ -0,0 +1,38 @@
+package diagnostics
+
+import (
+	"encoding/json"
+	"github.com/ledgerwatch/erigon-lib/common/dbg"
+	"net/http"
+	"time"
+)
+
+func SetupMemAccess(metricsMux *http.ServeMux) {
+	metricsMux.HandleFunc("/mem", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+		w.Header().Set("Content-Type", "application/json")
+		writeMem(w)
+	})
+
+	// update prometheus memory stats at least every 30 seconds
+	go func() {
+		ticker := time.NewTicker(30 * time.Second)
+		defer ticker.Stop()
+
+		for {
+			dbg.GetMemUsage()
```

Comment:
suggest to be explicit when swallowing an err:
```
_, _ := dbg.GetMemUsages() // err ignored since this is best effort background information logging
```

or 

```
_, err := dbg.GetMemUsage()
if err != nil {
    logger.Trace("...")
}
```